webknossos/examples/image_list_to_wkw.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+from webknossos import Dataset
+
+INPUT_FILES = (
+    Path("path/to/file1.jpg"),
```

Comment:
As discussed, please use some real microscopy images and add them as test data to the repository